File: PHP/Car.php
<?php
class Car {
    public $id;
    public $license;
    public $driver;
    public $passegenger;

    public function __construct($license, $driver){

        $this->license = $license;
        $this->driver = $driver;
        
    }

    public function printDataCar(){
        echo 'Driver Name: '.$this->license->name,' license: '.$this->license."\n";
    }
}

File: Java/Car.java
package Java;

class Car {
    Integer id;
    String license;
    Account driver;
    private Integer passenger;

    public Car(String license, Account driver){
        this.license = license;
        this.driver = driver;
    }

    void printDataCar(){
        System.out.println("License: " + this.license + "Driver name: "+ this.driver.name + "Passangers: " + this.passenger);
    }

    public Integer getPassenger(){
        return this.passenger;
    }

    public void setPassenger(Integer passenger){
        if(passenger == 4){
            this.passenger = passenger;
        }else{
            System.out.println("You need to set 4 passangers");
        }
    }
}
